Yoast/YoastSEO.js
# RW/move markers icons left

Fixes #622 

Testing:
1. Create new post
2. Write some test content
3. Check if the markers (eye-icons) show and correctly align.
4. Test the markers by clicking on them.
